rpcclient/evo_test.go

```diff
@@ -71,6 +81,14 @@ func TestQuorumInfo(t *testing.T) {
 	}
 	quorumHash := list.Llmq400_60[0]
 
+	client.httpClient.Transport = mockRoundTripperFunc(btcjson.QuorumInfoResult{
+		Height:          264072,
+		Type:            "llmq400_60",
+		QuorumHash:      "000004bfc56646880bfeb80a0b89ad955e557ead7b0f09bcc61e56c8473eaea9",
+		MinedBlock:      "000006113a77b35a0ed606b08ecb8e37f1ac7e2d773c365bd07064a72ae9a61d",
+		QuorumPublicKey: "0644ff153b9b92c6a59e2adf4ef0b9836f7f6af05fe432ffdcb69bc9e300a2a70af4a8d9fc61323f6b81074d740033d2",
+		SecretKeyShare:  "3da0d8f532309660f7f44aa0ed42c1569773b39c70f5771ce5604be77e50759e",
+	})
```

Comment:
`compareWithCliCommand` does nothing, it means we never verify data retrieved from `client.QuorumInfo`. I feel we should do that verification, for example add some `assert.Equal()` conditions.

Also applies to other tests that use `compareWithCliCommand()`.